metadig/checks.py

```
@@ -422,6 +419,10 @@ def select_nodes(context_node, selector_context):
                     value = False
                 else:
                     value = text_val
-        values.append(value)
-
+        # Convert a list to a single value
+        if isinstance(value, list):
+            if len(value) != 0:
+                values.append(str(value[0]))
```

Why do you silently drop other members of the list beyond index 0? Wouldn't you want to append those too?

---

I realized this mistake soon after pushing, and refactored this to use `.extend` which does not drop any members